src/utils/utils.js

```diff
@@ -35,8 +35,8 @@ export const getConfigObjectsForAnalytics = (configurations, groupCode) => {
     const numeratorsInGroup = configurations.numerators.filter((numerator) => {
         return (
             members.includes(numerator.code) &&
-            numerator.dataID !== null &&
-            numerator.dataSetID !== null
+            numerator.dataID &&
+            numerator.dataSetID
```

Comment:
Do you need to account for values that are empty arrays here (which would resolve truthy), or would `dataID` be falsy in that case (and the boolean expression would evaluate to `false` anyway)?

Reply:
Good point! I've added `&& numerator.dataSetID.length`

Without double checking the current code, I _think_ you shouldn't now actually end up with an empty array if you edit through the UI, but if someone edited the datastore directly they might end up in this case. For undefined/null. we're handling these mostly because the current configuration has examples of these (I haven't double checked if the old app actually resulted in these, or just someone made a configuration outside of the UI)